GAE/src/org/waterforpeople/mapping/dataexport/GraphicalSurveySummaryExporter.java

```diff
@@ -716,6 +723,32 @@ private synchronized int writeInstanceData(Sheet sheet, final int startRow,
 
         return maxRow + 1;
     }
+    
+    private String constructDisplayName(
```

Comment:
This function in essence does what `SurveyedLocale.assembleDisplayName()` does. Perhaps its worth making that one static and having the display name generation code in one place?